pyspeckit/spectrum/models/ammonia.py

```diff
@@ -176,44 +178,50 @@ def ammonia(xarr, tkin=20, tex=None, ntot=14, width=1, xoff_v=0.0,
         comment each step carefully.
         """
         Zpara = (2*Jpara+1)*np.exp(-h*(Brot*Jpara*(Jpara+1)+
-                                       (Crot-Brot)*Jpara**2)/(kb*tkin))
+                                       (Crot-Brot)*Jpara**2)/(kb*trot))
```

Comment:
:+1: 
In the sense that the (2,2) to (1,1) Trot applies to all the levels.  A fine assumption, but definitely a stated assumption